### PR TITLE
Added material/content for the Git session

### PR DESCRIPTION
**Pull request by lwjohnst86**
This is in prep for tomorrow's session. :smiley:

### CODE REVIEW

**Comment by QuLogic** [comment]
> git config --global core.editor "your_editor"

Does this require a full path on Windows or Mac?

**Comment by lwjohnst86** [comment]
@QuLogic it shouldn't. If you put like "notepad" or "textmate" or something similar, it should work just fine. I haven't had any problems yet with students. It seems to work. I would never anticipate problems on the Mac end, but maybe sometimes there might be problems on the Windows end.

File: lessons/git/lesson.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+For configuring your git, follow the "Initial setup" I've put
+together:
+
+* http://codeasmanuscript.github.io/workshops/lessons/git/cheatsheet/
+
+Commands:
+
+    cd ~/Desktop
+    git clone https://github.com/lwjohnst86/studyGroup.git
```
Comment (QuLogic) [inline lessons/git/lesson.md:42]: Yours or @mbonsma's?

**Comment by QuLogic** [comment]
@mbonsma If we had that organization, we could have created a throwaway repo for students to play around in, instead of messing with the "main" one.

**Comment by lwjohnst86** [comment]
@QuLogic no worries, they can't mess with the main one unless @mbonsma allows them to have access. That's why I will be doing forks and pull requests. We can always reject them. I think it's nice because it gives the learners a sense of context in a real-world situation.

File: lessons/git/lesson.md (hunk 1 above)
Comment (lwjohnst86) [inline lessons/git/lesson.md:42]: Good point, I was going to fix that. Will do that.

**Comment by lwjohnst86** [comment]
@QuLogic here is the pull request with the changes you highlighted: https://github.com/mbonsma/studyGroup/pull/9

**Comment by QuLogic** [comment]
I noticed... :stuck_out_tongue:

**Comment by mbonsma** [comment]
@QuLogic, let's talk tomorrow about the organization part. Part of me wants to give it a week or two to make sure anyone who may have followed an old link in the emails I sent can still find us. Unless there's a way to make it redirect...?

**Comment by QuLogic** [comment]
[Redirects](https://help.github.com/articles/transferring-a-repository/#redirects-and-git-remotes)

> When a repository is transferred, all links to the previous location are automatically redirected to the new location.

**Comment by mbonsma** [comment]
Amazing. Okay, let's do it! @QuLogic, are you willing to dispense advice on this tomorrow, maybe after the meetup? I'm a little nervous doing it myself.

**Comment by QuLogic** [comment]
Sure, I don't think it would take too much work, though.

Actually, I forgot you already created an organization. Maybe we should create a temporary repo there for the tutorial to keep this one clean(ish). @lwjohnst86 can decide since he's running it.

**Comment by lwjohnst86** [comment]
@QuLogic @mbonsma I wouldn't mind creating a temporary repo in this organization for the lesson, but I'm not a member, nor have permission. We could do this a bit before the session.

**Comment by mbonsma** [comment]
I'll plan to be in MP408 by about 5:45.

**Pull request merged**